templates/rss.ftl

```diff
@@ -13,6 +13,9 @@
   <category>Keycloak/SSO/Identity and Access Management</category>
   <#list blogs as blog>
     <#assign description>
+        <#list blog.getMap() as key, value>
+            <@"<#assign ${key}=value>"?interpret />
+        </#list>
```

Comment:
This doesn't make any sense to me; can you explain this bit?

Reply:
This makes the fields (e.g. `source`) that are needed in the release blog template available to the template that is included in the lines after this section. Other builders render the templates explicitly, passing a single `blog` plus the values from `getMap()`, but the RSS builder passes the `blogs` from the root, so some extra work needs to be done to expose those variables.

 AFAIK there is no other way to make variables that have a dynamic name available otherwise. Although quite hacky, this is what the [FreeMarker documentation recommends](https://freemarker.apache.org/docs/app_faq.html#faq_assign_to_dynamic_variable_name).

Reply:
Ah, okay seems to work, so fine with it